storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedAppendBlobs.java

```diff
@@ -35,7 +35,6 @@
 import java.net.URL;
 import java.nio.ByteBuffer;
 import java.time.OffsetDateTime;
-import java.util.HashMap;
```

Comment:
This file and the other files prefixed with Generated are also output from Autorest, so they should also be excluded from the checkstyle report.

Reply:
Good catch